cmd/software/deploy.go

```diff
@@ -51,6 +52,7 @@ func NewDeployCmd() *cobra.Command {
 	cmd.Flags().BoolVarP(&saveDeployConfig, "save", "s", false, "Save deployment in config for future deploys")
 	cmd.Flags().BoolVarP(&ignoreCacheDeploy, "no-cache", "", false, "Do not use cache when building container image")
 	cmd.Flags().StringVar(&workspaceID, "workspace-id", "", "workspace assigned to deployment")
+	cmd.Flags().BoolVarP(&isDagOnlyDeploy, "dag-only", "", false, "Deploy only those dags which are present in the /dags folder")
```

Comment:
The flag comment does not seem inline with what's the actual behaviour of the command, i.e. we don't seem to be deploying any DAGs

Reply:
I will work on it now and will create a new PR for it. Should I add a todo comment?
The idea is to create small PRs rather than a big one @neel-astro

Reply:
Added the software version check